coordinator/clientapi/clientapi.go

```diff
@@ -741,3 +758,41 @@ func (a *ClientAPI) WriteSecrets(ctx context.Context, rawSecretManifest []byte,
 
 	return commit(ctx)
 }
+
+// SignQuote verifies the quote and signs it with the Coordinator's root key.
+func (a *ClientAPI) SignQuote(ctx context.Context, quote []byte) (signedQuote []byte, tcbStatus string, err error) {
+	a.log.Info("SignQuote called")
+	defer func() {
+		if err != nil {
+			a.log.Error("SignQuote failed", zap.Error(err))
+		} else {
+			a.log.Info("SignQuote successful")
+		}
+	}()
+
+	// Verify the quote
+	var verifyErr error
+	report, err := enclave.VerifyRemoteReport(quote)
+	if err != nil {
```

Comment:
```suggestion
	report, err := enclave.VerifyRemoteReport(quote)
	if err != nil && !errors.Is(err, attestation.ErrTCBLevelInvalid) {
```
also consider adding a unit test for this func